spec/index.bs

```diff
@@ -3292,10 +3292,25 @@ Issue(https://github.com/gpuweb/gpuweb/issues/693): add a limit to the number of
 
 <script type=idl>
 interface GPUCommandBuffer {
+    readonly attribute Promise<double> executionTime;
 };
 GPUCommandBuffer includes GPUObjectBase;
 </script>
 
+{{GPUCommandBuffer}} has the following attributes:
+
+<dl dfn-type=attribute dfn-for="GPUCommandBuffer">
+    : <dfn>\[[executionTime]]</dfn> of type Promise<{{double}}>, readonly
+    ::
+        The total time, in seconds, that GPU execute this command buffer.
+
+        - If {{GPUCommandEncoderDescriptor/measureExecutionTime}} is set to true:
+
+            - |promise| [=resolves=] with the execution time of command buffer. The |promise| of execution time is fecthed when GPUQueue.{{GPUQueue/submit()}}.
```

Comment:
This is not an algorithm so these variables (`|promise|`) don't take effect. Fixing this.